test/services/utils/IssueUtil.test.ts

```diff
@@ -0,0 +1,213 @@
+import {
+  findSigLabel,
+  isChallengeIssue,
+  needHelp,
+  checkIsInAssignFlow,
+  isClosed,
+  findMentorAndScore,
+} from "../../../src/services/utils/IssueUtil";
+import {
+  CHALLENGE_PROGRAM_LABEL,
+  HELP_WANTED_LABEL,
+} from "../../../src/commands/labels";
+
+describe("Issue Util", () => {
+  test("issue have sig label", () => {
+    expect(
+      findSigLabel([
```

Comment:
I think that after calling the function, storing the return value through a variable and then asserting may be more readable.